helpers/jestHelper.js

```diff
@@ -0,0 +1,27 @@
+/* global it: true */
+/* eslint strict:[2,"function"] */
+var jsc = require("jsverify");
+
+/**
+ * wrapper function for using jsverify with jest syntax
+ *
+ * @param description string describing the test
+ * @param arbitrary a jsverify record (or other arbitrary)
+ * @param testFn function that expects, in proper jest syntax
+ * @param opts options for jsverify
+ */
+function itHolds(description, arbitrary, testFn, options) {
```

Comment:
single arbitrary only?

Reply:
My reworked version supports everything that `jsc.property` supports since it just wraps that.